test/TestImport.hs

```diff
@@ -55,3 +56,20 @@ getTables = do
     |] []
 
     return $ map unSingle tables
+
+-- | Authenticate as a user. This relies on the `development: true` flag being set in test-settings.yaml,
```

Comment:
I think this comment is outdated now and this relies on `appAuthDummyLogin` now ?